### ISSUE TITLE
Suite Number not parsed

### ISSUE THREAD

**Issue by chuckdelcamp** [datamade/usaddress#133]
When a suite, office, or apartment number is not labeled as such, it is not identified using usaddress.tag().

Example:
ORIGINAL STRING:  14855 Blanco Rd.  #108 None San Antonio TX 78216
PARSED TOKENS:    [(u'14855', 'AddressNumber'), (u'Blanco', 'StreetName'), (u'Rd.', 'StreetNamePostType'), (u'#', 'OccupancyIdentifier'), (u'108', 'AddressNumber'), (u'None', 'StreetName'), (u'San', 'PlaceName'), (u'Antonio', 'PlaceName'), (u'TX', 'StateName'), (u'78216', 'ZipCode')]
UNCERTAIN LABEL:  AddressNumber

When this error is raised, it's likely that either (1) the string is not a valid person/corporation name or (2) some tokens were labeled incorrectly

This will be parsed using usaddress.parse(), however the 'OccupancyIdentifier' will be found as '#'.

**Comment by cathydeng** [comment]
interesting - thanks for reporting. do you have a few more examples of address strings like this, so that we can add 'em to the training data?

**Comment by chuckdelcamp** [comment]
Will do.
Found another, when a city name is three words long, the first word is treated as a street:

ORIGINAL STRING:  800 BIESTERFIELD RD STE 402 ELK GROVE VILLAGE IL 60007
PARSED TOKENS:    [(u'800', 'AddressNumber'), **(u'BIESTERFIELD', 'StreetName'),** (u'RD', 'StreetNamePostType'), (u'STE', 'OccupancyType'), (u'402', 'OccupancyIdentifier'), **(u'ELK', 'StreetName'),** (u'GROVE', 'StreetName'), (u'VILLAGE', 'StreetNamePostType'), (u'IL', 'StateName'), (u'60007', 'ZipCode')]
UNCERTAIN LABEL:  StreetName

**Comment by jeancochrane** [comment]
Hey @chuckdelcamp,

I'm bringing these in for the next round of training data, but I'm a little unsure about your first example. Should "None" be included in the address string? It looks like a data entry error. When I remove it I get the expected behavior:

```python
In [6]: usaddress.tag('14855 Blanco Rd. #108 San Antonio TX 78216')

Out[6]:
(OrderedDict([('AddressNumber', '14855'),
              ('StreetName', 'Blanco'),
              ('StreetNamePostType', 'Rd.'),
              ('OccupancyIdentifier', '# 108'),
              ('PlaceName', 'San Antonio'),
              ('StateName', 'TX'),
              ('ZipCode', '78216')]),
 'Street Address') 
```

I added your second example in – if you have any more examples, send 'em our way!

**Comment by jeancochrane** [comment]
Fixed in [v0.5.8.](https://github.com/datamade/usaddress/commit/407ef238903c9c65da652c288af5990dc412773a)

New behavior:

```python
In [6]: usaddress.tag("14855 Blanco Rd. #108 None San Antonio TX 78216")
Out[6]:
(OrderedDict([('AddressNumber', '14855'),
              ('StreetName', 'Blanco'),
              ('StreetNamePostType', 'Rd.'),
              ('OccupancyIdentifier', '# 108'),
              ('PlaceName', 'None San Antonio'),
              ('StateName', 'TX'),
              ('ZipCode', '78216')]),
 'Street Address')

In [7]: usaddress.tag(" 800 BIESTERFIELD RD STE 402 ELK GROVE VILLAGE IL 60007")
Out[7]:
(OrderedDict([('AddressNumber', '800'),
              ('StreetName', 'BIESTERFIELD'),
              ('StreetNamePostType', 'RD'),
              ('OccupancyType', 'STE'),
              ('OccupancyIdentifier', '402'),
              ('PlaceName', 'ELK GROVE VILLAGE'),
              ('StateName', 'IL'),
              ('ZipCode', '60007')]),
 'Street Address')
```

Get the latest release via pip:

```bash
pip install --upgrade usaddress
```
